src/Components/Server/src/Builder/EndpointBuilderComponentExtensions.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Components.Server;
+
+namespace Microsoft.AspNetCore.Builder
+{
+    /// <summary>
+    /// Extensions for <see cref="IEndpointConventionBuilder"/>.
+    /// </summary>
+    public static class EndpointConventionBuilderComponentsExtensions
```

Comment:
Consider renaming this to `ComponentEndpointConventionBuilderExtensions` for consistency.

Reply:
super nit pick tho

Reply:
Done